src/ng/rootScope.js

```diff
@@ -1374,3 +1376,6 @@ function $RootScopeProvider() {
     }
   }];
 }
+
+$RootScopeProvider.$inject = ['$shutdownProvider'];
```

Comment:
IMO, it is preferrable to have the "DI annotations" above the function, so it is easier to update both (the annotation and the function arguments).